.github/workflows/ci.yml

```diff
@@ -16,6 +16,7 @@ jobs:
       CC: "clang"
       CXX: "clang++"
       npm_config_clang: "1"
+      MAKEFLAGS: -j4
```

Comment:
Is this needed globally? I see the flag is set inside `alpine-release`, which should make this unnecessary if it's only needed for Alpine builds...

Reply:
Removed